### PR TITLE
feat: wallet withdraw test

### PR DESCRIPTION
**Pull request by notanatol**

### CODE REVIEW

File: pkg/check/withdraw/withdraw.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+		return err
+	}
+
+	target := checkCase.Bee(1)
+
+	c.logger.Infof("target is %s", target.Name())
+
+	c.logger.Info("withdrawing native...")
+
+	if err := target.Withdraw(ctx, "NativeToken", o.TargetAddr); err != nil {
```
Comment (istae) [inline pkg/check/withdraw/withdraw.go:58]: the before and after balances should be compared after the withdraw

**Review by istae — CHANGES_REQUESTED** [review]

File: pkg/check/withdraw/withdraw.go (hunk 1 above)
Comment (notanatol) [reply pkg/check/withdraw/withdraw.go:58]: it's being tested inside the function, this way I can use the same method for different coin/addresses:

https://github.com/ethersphere/beekeeper/pull/386/files#diff-26ffebd2abeeaff0e1d88eab9a47503856f31fcfc6fd6704b431ea5c4b72727cR125

File: pkg/bee/client.go [hunk 2]
```diff
@@ -847,3 +848,33 @@ func (c *Client) GetStake(ctx context.Context) (*big.Int, error) {
 func (c *Client) WithdrawStake(ctx context.Context) (string, error) {
 	return c.debug.Stake.WithdrawStake(ctx)
 }
+
+// WithdrawStake withdraws stake
```
Comment (acha-bill) [inline pkg/bee/client.go:852]: Update comment

File: pkg/bee/debugapi/node.go [hunk 3]
```diff
@@ -249,3 +251,29 @@ func (n *NodeService) Topology(ctx context.Context) (resp Topology, err error) {
...
+
+// Wallet returns the wallet state
+func (n *NodeService) Wallet(ctx context.Context) (resp Wallet, err error) {
+	err = n.client.requestJSON(ctx, http.MethodGet, "/wallet", nil, &resp)
+	return
+}
+
+// Withdraw calls wallet withdraw endpoint
+func (n *NodeService) Withdraw(ctx context.Context, token, addr string) (tx common.Hash, err error) {
+	endpoint := fmt.Sprintf("/wallet/withdraw/%s?address=%s&amount=1000000", token, addr)
```
Comment (acha-bill) [inline pkg/bee/debugapi/node.go:268]: why hard code amount here?

Comment (notanatol) [reply pkg/bee/debugapi/node.go:268]: it's unlikely to change so didn't want to make it configurable

Comment (notanatol) [reply pkg/bee/debugapi/node.go]: moved it to a constant on the callers side

**Review by acha-bill — APPROVED** [review]

**Review by istae — APPROVED** [review]

**Pull request merged**